### PR TITLE
Port to project.json/xproj and add .NET Standard support

### PR DESCRIPTION
**Pull request by jeremymeng**

### CODE REVIEW

File: src/Serilog.Sinks.NLog/project.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "2.0.0-*",
+  "description": "Serilog event sink that writes to NLog. Merge your new Serilog event stream into your existing NLog infrastructure.",
```
Comment (jeremymeng) [inline src/Serilog.Sinks.NLog/project.json:3]: I left the "-*" part here since NLog v4.4.0 is still in beta.

File: Build.ps1 [hunk 2]
```diff
@@ -1,85 +1,15 @@
-param(
-    [String] $majorMinor = "0.0",  # 2.0
-    [String] $patch = "0",         # $env:APPVEYOR_BUILD_VERSION
-    [String] $customLogger = "",   # C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
-    [Switch] $notouch,
-    [String] $sln                  # e.g serilog-sink-name
-)
+Push-Location $PSScriptRoot
```
Comment (nblumhardt) [inline Build.ps1:3]: This script is slightly older than the one in serilog/serilog (there were some issues with build number generation). Can you please update with the latest Build.ps1 from the main Serilog repo?

File: src/Serilog.Sinks.NLog/project.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "2.0.0-*",
```
Comment (nblumhardt) [inline src/Serilog.Sinks.NLog/project.json]: There's already a 2.x of this sink published - can we update this to 3.0.0-\* please?

**Comment by nblumhardt** [comment]
Thanks Jeremy, this is looking great. Can you please re-target the PR to the `dev` branch? I just noticed this repo is missing one, so I've created it now :-)

File: Build.ps1 (hunk 2 above)
Comment (jeremymeng) [inline Build.ps1:3]: Thank you @nblumhardt for reviewing this.

I see there are [test sections](https://github.com/serilog/serilog/blob/dev/Build.ps1#L29-L49) which don't apply to this repro.  I could either add a check for the existence of `test` folder, or remote the sections for this repo.  Which approach do you prefer?

Comment (nblumhardt) [inline Build.ps1:3]: The existence check would be fine; where possible I've tried adding empty test projects in these cases, which is probably the ideal option if it's not too much trouble.

**Comment by jeremymeng** [comment]
Closing this, will open a new one targeting `dev` branch.

**Comment by nblumhardt** [comment]
Thanks. Just a heads-up, the `Edit` button next to the title in a PR now lets you change the target branch without closing/reopening.

**Comment by jeremymeng** [comment]
Ah didn't know that.  Thanks a lot for the info!

**Pull request closed without merging**